docs/src/python/user-guide/misc/visualization.py

```diff
@@ -0,0 +1,141 @@
+# --8<-- [start:setup]
+import base64
+
+import polars as pl
+
+url = "docs/data/iris.csv"
+
+# --8<-- [end:setup]
+
+# --8<-- [start:dataframe]
+
+iris_agg_df = (
+    pl.scan_csv(url)
+    .group_by("species")
+    .agg(
+        pl.col("petal_length").mean(),
+    )
+    .collect()
+)
+
+# --8<-- [end:dataframe]
+
+# --8<-- [start:hvplot_make_plot]
+import hvplot
+
+plot = iris_agg_df.plot.bar(
+    x="species",
+    y="petal_length",
+    width=650,
+)
+hvplot.save(plot, "docs/images/hvplot_bar.html")
+with open("docs/images/hvplot_bar.html", "r") as f:
+    chart_html = f.read()
+    print(f"{chart_html}")
+
+# --8<-- [end:hvplot_make_plot]
+"""
+# --8<-- [start:hvplot_show_plot]
+iris_agg_df.plot.bar(
+    x="species",
+    y="petal_length",
+    width=650,
+)
+# --8<-- [end:hvplot_show_plot]
+"""
+
+
+# --8<-- [start:matplotlib_make_plot]
+import matplotlib.pyplot as plt
+
+plt.bar(x=iris_agg_df["species"], height=iris_agg_df["petal_length"])
+plt.savefig("docs/images/matplotlib_bar.png")
+with open("docs/images/matplotlib_bar.png", "rb") as f:
+    png = base64.b64encode(f.read()).decode()
+    print(f'<img src="data:image/png;base64, {png}"/>')
+
+# --8<-- [end:matplotlib_make_plot]
+"""
+# --8<-- [start:matplotlib_show_plot]
+plt.bar(x=iris_agg_df["species"], height=iris_agg_df["petal_length"])
+# --8<-- [end:matplotlib_show_plot]
+"""
+# --8<-- [start:seaborn_make_plot]
+import seaborn as sns
+
+sns.barplot(
+    iris_agg_df,
+    x="species",
+    y="petal_length",
+)
+plt.savefig("docs/images/seaborn_bar.png")
+with open("docs/images/seaborn_bar.png", "rb") as f:
+    png = base64.b64encode(f.read()).decode()
+    print(f'<img src="data:image/png;base64, {png}"/>')
+
+# --8<-- [end:seaborn_make_plot]
+"""
+# --8<-- [start:seaborn_show_plot]
+sns.barplot(
+    iris_agg_df,
+    x="species",
+    y="petal_length",
+)
+# --8<-- [end:seaborn_show_plot]
+"""
+
+# --8<-- [start:plotly_make_plot]
+import plotly.express as px
+
+fig = px.bar(
+    iris_agg_df,
+    x="species",
+    y="petal_length",
+    width=650,
+)
+fig.write_html("docs/images/plotly_bar.html", full_html=False, include_plotlyjs="cdn")
+with open("docs/images/plotly_bar.html", "r") as f:
+    chart_html = f.read()
+    print(f"{chart_html}")
+
+# --8<-- [end:plotly_make_plot]
+"""
+# --8<-- [start:plotly_show_plot]
+px.bar(
+    iris_agg_df,
+    x="species",
+    y="petal_length",
+    width=400,
+)
+# --8<-- [end:plotly_show_plot]
+"""
+# --8<-- [start:altair_make_plot]
+
+import altair as alt
```

Comment:
I don't see these either

![image](https://github.com/pola-rs/polars/assets/33491632/1c525ba0-79ed-451e-bf90-4f24900cb651)

Reply:
@MarcoGorelli Can you confirm what files you have in docs/images please?

Reply:
just pulled and tried again and I can see them - resolving then, thanks!